tests/subprocesstest.py

```diff
@@ -0,0 +1,118 @@
+"""
+subprocesstest enables unittest test cases and suites to be run in separate
+python interpreter instances.
+
+A base class SubprocessTestCase is provided that, when extended, will run test
+cases marked with @run_in_subprocess in a separate python interpreter.
+"""
+import os
+import subprocess
+import sys
+import unittest
+
+
+SUBPROC_TEST_ATTR = '_subproc_test'
+SUBPROC_ENV_VAR = '_SP_TEST'
```

Comment:
Instead of a "private" env variable, I'd prefer something more clear that we could supply manually when running. `SP_TEST` is fine, `SUBPROCESS_TEST`, etc, just prefer no leading `_`.